<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>275</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>502</width>
    <height>275</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0" colspan="2">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QRadioButton" name="m_deg">
          <property name="styleSheet">
           <string notr="true">color: rgb(135, 195, 211);</string>
          </property>
          <property name="text">
           <string>deg</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLCDNumber" name="output">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
            <stylestrategy>PreferDefault</stylestrategy>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);
background-color: rgb(100, 164, 193);
color: rgb(255, 255, 255);</string>
          </property>
          <property name="smallDecimalPoint">
           <bool>true</bool>
          </property>
          <property name="digitCount">
           <number>30</number>
          </property>
          <property name="mode">
           <enum>QLCDNumber::Dec</enum>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
          <property name="value" stdset="0">
           <double>0.000000000000000</double>
          </property>
          <property name="intValue" stdset="0">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="exp">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);
</string>
          </property>
          <property name="text">
           <string>π</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="seven">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="eight">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="nine">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="mult">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="koren">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>√</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ystepx">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>yˣ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="minsert">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>MR</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="mdel">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>MC</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="minus">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>M-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="mplus">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>M+</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="second">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>2nd</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sin">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>sin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cos">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>cos</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="tg">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>tg</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="log">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>log</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ln">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>ln</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="m_rad">
          <property name="styleSheet">
           <string notr="true">color: rgb(135, 195, 211);</string>
          </property>
          <property name="text">
           <string>rad</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="quadrat">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string notr="true">x²</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="openSK">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>(</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="closeSK">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="tab">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>-&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="click">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>*click*</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="sbros_str">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>CE</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="one">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="two">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="three">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="plus">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="8" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="sbros_all">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>C</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="nol">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="koma">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="minus_2">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="print">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>=</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="plus_or_minus">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>+/-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Mista_trigger">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="five">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="six">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delit">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(126, 182, 197);</string>
          </property>
          <property name="text">
           <string>÷</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
